test/examples/DeFi/MockFlashLoan.sol

```diff
@@ -10,7 +11,7 @@ interface IFlashLoanBorrower {
         uint256 amount1,
         address token2,
         uint256 amount2,
-        bytes calldata data
+        CallObject[] calldata callObjs
```

Comment:
This change is not needed, in real use case it will be a third party contract

Reply:
Ok will use bytes instead of CallObj